schema/deploy/tables/product.sql

```diff
@@ -71,6 +75,10 @@ comment on column ggircs_portal.product.description is 'The description of the p
 comment on column ggircs_portal.product.units is 'The unit of measure for the product';
 comment on column ggircs_portal.product.requires_emission_allocation is 'Boolean value indicates if the product requires allocation of emissions';
 comment on column ggircs_portal.product.is_ciip_product is 'Boolean value indicates if the product is benchmarked and has an associated incentive';
+comment on column ggircs_portal.product.includes_imported_electricity is 'Boolean value indicates if the product should attribute imported electricity to emissions';
+comment on column ggircs_portal.product.includes_exported_electricity is 'Boolean value indicates if the product should attribute exported electricity to emissions';
+comment on column ggircs_portal.product.includes_imported_heat is 'Boolean value indicates if the product should attribute imported heat to emissions';
+comment on column ggircs_portal.product.includes_exported_heat is 'Boolean value indicates if the product should attribute exported heat to emissions';
```

Comment:
Those can be more explicit: `add_imported_electricity_emissions` could be 'Boolean value indicates if imported electricity emissions should be added to the facility emissions when calculating the product emission intensity (applies only to products where requires_emission_allocation is false)'